visual-dotnet/SauceLabs.Visual/GraphQL/FullPageConfigIn.cs

```diff
@@ -7,6 +8,6 @@ internal class FullPageConfigIn
         [JsonProperty("delayAfterScrollMs")]
         public int? DelayAfterScrollMs { get; set; }
         [JsonProperty("hideAfterFirstScroll")]
-        public string[]? HideAfterFirstScroll { get; set; }
+        public IEnumerable? HideAfterFirstScroll { get; set; }
```

Comment:
I think that's preferable to use generics with `IEnumerable`, so we know the type that is in it.
```suggestion
        public IEnumerable<string>? HideAfterFirstScroll { get; set; }
```